lib/Doctrine/Validator/Notblank.php

```diff
@@ -41,6 +41,6 @@ class Doctrine_Validator_Notblank extends Doctrine_Validator_Driver
      */
     public function validate($value)
     {
-        return (trim($value) !== '' && $value !== null);
+        return (null !== $value && '' !== trim($value));
```

Comment:
🤪

Reply:
This is an upstream fix to avoid `trim()` throwing a fit because of a `null` argument, wrapped up in one of your Spaceballs syntaxes.